functions.sh

```diff
@@ -61,4 +61,21 @@ function saveTaskStatus() {
   fi
 }
 
+function decrypt_credentials() {
+    local fernet_key="$1"
+    local encrypted_value="$2"
 
+    python3 -c "
```

Comment:
If this is a python code why we are trying to solve it via bash

Reply:
We are not comfortable with python so bash is what we'll proceed with.